### PR TITLE
fix(prisma-client-reference): Remove misleading `distinct` options for count and aggreagate (+ all kinds of things along the way)

### PR DESCRIPTION
**Pull request by janpio**
- Remove `disctinct` from `count` and `aggregate` options/reference

Also:
- Remove `select` from `count` options/reference
- Resort tables of options to match types (select,include,where => orderBy => cursor/take/skip => disctinct usually) and logical priority (= required before optional)
- Fix "Remarks" headline level of "rejectOnNotFound"
- Fix broken Options table for `groupBy`
- Fix code highlighting for `create` example

### CODE REVIEW

File: content/400-reference/200-api-reference/050-prisma-client-reference.mdx [hunk 1]
```diff
@@ -1453,7 +1452,6 @@ export type UserFindManyArgs = {
   cursor?: UserWhereUniqueInput
   take?: number
   skip?: number
-  distinct?: Enumerable<UserDistinctFieldEnum>
```
Comment (janpio) [inline content/400-reference/200-api-reference/050-prisma-client-reference.mdx:1456]: main change 1

File: content/400-reference/200-api-reference/050-prisma-client-reference.mdx [hunk 2]
```diff
@@ -1536,11 +1534,10 @@ See also: [Aggregation, grouping, and summarizing](/concepts/components/prisma-c
 ```ts file=index.d.ts
 export type UserAggregateArgs = {
   where?: UserWhereInput
-  orderBy?: XOR<Enumerable<UserOrderByInput>, UserOrderByInput>
+  orderBy?: UserOrderByWithRelationInput | UserOrderByWithRelationInput[]
   cursor?: UserWhereUniqueInput
   take?: number
   skip?: number
-  distinct?: Enumerable<UserDistinctFieldEnum>
```
Comment (janpio) [inline content/400-reference/200-api-reference/050-prisma-client-reference.mdx:1543]: main change 2

File: content/400-reference/200-api-reference/050-prisma-client-reference.mdx [hunk 3]
```diff
@@ -624,14 +624,14 @@ We introduced `findUniqueOrThrow` in v4.0.0. It replaces the [`rejectOnNotFound`
...
 | `take`                          | `number`                                                     | No       | Specifies how many objects should be returned in the list. When used with `findFirst`, `take` is implicitly `1` or `-1`. `findFirst` is only affected by whether the value is positive or negative - any negative value reverses the list.                      |
+| `skip`                          | `number`                                                     | No       | Specifies how many of the returned objects in the list should be skipped.                                                                                                                                                                                       |
+| `distinct`                      | `Enumerable<UserDistinct`<br />`FieldEnum>`                  | No       | Lets you filter out duplicate rows by a specific field - for example, return only distinct `Post` titles.                                                                                                                                                       |
```
Comment (janpio) [inline content/400-reference/200-api-reference/050-prisma-client-reference.mdx:634]: resort

File: content/400-reference/200-api-reference/050-prisma-client-reference.mdx [hunk 4]
```diff
@@ -745,13 +745,13 @@ We introduced `findFirstOrThrow` in v4.0.0. It replaces the [`rejectOnNotFound`]
...
 | `take`     | `number`                                                      | No       | Specifies how many objects should be returned in the list (as seen from the _beginning_ (positive value) or _end_ (negative value) **either** of the list **or** from the `cursor` position if mentioned) |
-| `select`   | `XOR<PostSelect, null>`                                       | No       | [Specifies which properties to include](/concepts/components/prisma-client/select-fields) on the returned object.                                                                                         |
-| `include`  | `XOR<PostInclude, null>`                                      | No       | [Specifies which relations should be eagerly loaded](/concepts/components/prisma-client/relation-queries) on the returned object.                                                                         |
+| `skip`     | `number`                                                      | No       | Specifies how many of the returned objects in the list should be skipped.                                                                                                                                 |
```
Comment (janpio) [inline content/400-reference/200-api-reference/050-prisma-client-reference.mdx:754]: resort

File: content/400-reference/200-api-reference/050-prisma-client-reference.mdx [hunk 5]
```diff
@@ -1428,11 +1428,10 @@ See [Filter conditions and operators](#filter-conditions-and-operators) for exam
...
 | `take`    | `number`                                                      | No       | Specifies how many objects should be returned in the list (as seen from the _beginning_ (positive value) or _end_ (negative value) **either** of the list **or** from the `cursor` position if mentioned) |
-| `orderBy` | `XOR<Enumerable<PostOrder`<br />`ByInput>, PostOrderByInput>` | No       | Lets you order the returned list by any property.                                                                                                                                                         |
-| `select`  | `UserCountAggregateInputType`                                 | No       | Select which fields to count (non-`null` values) - you can also count `_all`.                                                                                                                             |
+| `skip`    | `number`                                                      | No       | Specifies how many of the returned objects in the list should be skipped.                                                                                                                                 |
```
Comment (janpio) [inline content/400-reference/200-api-reference/050-prisma-client-reference.mdx:1434]: resort + remove wrong `select`

**Review by jharrell — APPROVED** [review]

**Pull request merged**